src/lib/segmentAnalytics.js

```diff
@@ -1,25 +1,45 @@
-export const trackClickForSegmentAnalytics = ({
-  type,
-  clickUrl,
-  clickText,
-  location,
-  userId,
+export const trackPageViewForSegment = ({
+  name,
+  path,
+  userExperience
 }) => {
   if (window.analytics) {
-    window.analytics.track(`CTA ${type} Clicked`, {
-      ...(clickUrl && { click_url: clickUrl }),
-      ...(clickText && { click_text: clickText }),
-      ...(location && { location }),
-      ...(userId && { user_id: userId }),
+    window.analytics.page("Page viewed", name, {
+      ...(path && { path: path }),
+      ...(userExperience && { user_experience: userExperience })
     });
   }
 };
 
-export const trackPageForSegmentAnalytics = ({ name, path, userId }) => {
+export const trackClickForSegment = ({
+  eventName,
+  clickType,
+  userExperience,
```

Comment:
@TrofimovAnton85 Just out of curiousity what info does this provide in the analytics. Does it give us insight into loading metrics, or customer satisfaction, etc...?

Reply:
@joaniefromtheblock  Hey! 
We have several types of events/metrics (they are described in the ticket) that we would like to collect for analytics. Based on them, we can test: what percentage of users use our updated reference pages and how convenient they are for them (do they use the parameter configurator, sending requests).

Reply:
sorry I meant specifically the `userExperience`?

Reply:
First, we plan to release updated pages for 10% of users (based on the feature flag condition). For them, this `userExperience` metric will take the value “new”, and the remaining 90% users => old pages will be displayed.